docs/apu_flashing_with_rpi.md

```diff
@@ -0,0 +1,82 @@
+1. Preparing coreboot repository:
```

Comment:
@BartekLew please use "View"  button to check how this is rendered to html - syntax have to be improved. Please add h2 to each point and fix backticks so those will cover only code.